Makefile

```diff
@@ -152,9 +158,94 @@ unit-test: go-install
 coverage: unit-test
 	./build/scripts/coverage.sh html
 
+#
+# Tests-related tasks
+#
+CLUSTER_NAME = verrazzano-operator
+CERTS = build/verrazzano-operator-cert
+VERRAZZANO_NS = verrazzano-system
+DEPLOY = build/deploy
+K8RESOURCE = test/k8resource
+
 .PHONY: integ-test
-integ-test: go-install
-	$(GO) test -v ./test/integ/ -timeout 30m --kubeconfig=${KUBECONFIG} --namespace=${K8S_NAMESPACE} --runid=${INTEG_RUN_ID}
+integ-test: create-cluster
+
+	echo 'Create CRDs needed by the verrazzano-operator...'
+	kubectl create -f vendor/${CRDGEN_PATH}/${CRD_PATH}/verrazzano.io_verrazzanomanagedclusters_crd.yaml
+	kubectl create -f vendor/${CRDGEN_PATH}/${CRD_PATH}/verrazzano.io_verrazzanomodels_crd.yaml
+	kubectl create -f vendor/${CRDGEN_PATH}/${CRD_PATH}/verrazzano.io_verrazzanobindings_crd.yaml
+	kubectl create -f vendor/${COH_PATH}/${CRD_PATH}/verrazzano.io_cohclusters_crd.yaml
+	kubectl create -f vendor/${HELIDON_PATH}/${CRD_PATH}/verrazzano.io_helidonapps_crd.yaml
+	kubectl create -f vendor/${WKO_PATH}/${CRD_PATH}/verrazzano.io_wlsoperators_crd.yaml
+	kubectl create -f vendor/${VMI_PATH}/${VMI_CRD_PATH}/verrazzano-monitoring-operator-crds.yaml
+
+	echo 'Deploy local cluster and required secret ...'
+	# Create the local cluster secret with empty kubeconfig data.  This will force the verrazzano-operator
+	# to use the in-cluster kubeconfig to access the managed cluster.
+	kubectl apply -f ./test/k8resource/local-cluster-secret.yaml
+	kubectl apply -f ${K8RESOURCE}/local-vmc.yaml
+
+	echo 'Load docker image for the verrazzano-operator...'
+	kind load docker-image --name ${CLUSTER_NAME} ${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
+
+	echo 'Load docker image for the linux slim used for fake micro operators ...'
+	docker pull container-registry.oracle.com/olcne/istio_kubectl:1.4.6
```

Comment:
Why do we need istio_kubectl image and the istio CRDs YAML?

Reply:
If I recall correctly, the job(s) that install the istio CRDs need istio_kubectl

Reply:
That image is needed by the istio CRD jobs.  The generated yaml requires kubectl image name so i tagged it